src/main/java/com/scalar/cassy/CassyClient.java

```diff
@@ -0,0 +1,115 @@
+package com.scalar.cassy;
+
+import com.scalar.cassy.rpc.BackupListingRequest;
+import com.scalar.cassy.rpc.BackupListingResponse;
+import com.scalar.cassy.rpc.BackupRequest;
+import com.scalar.cassy.rpc.BackupResponse;
+import com.scalar.cassy.rpc.CassyGrpc;
+import io.grpc.Channel;
+import io.grpc.ManagedChannelBuilder;
+import java.util.Arrays;
+import java.util.Optional;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.stream.Collectors;
+
+public class CassyClient {
+  private final CassyGrpc.CassyBlockingStub blockingStub;
+
+  public CassyClient() {
+    this(ManagedChannelBuilder.forAddress("localhost", 20051).usePlaintext());
+  }
+
+  public CassyClient(ManagedChannelBuilder<?> channelBuilder) {
+    Channel channel = channelBuilder.build();
+    blockingStub = CassyGrpc.newBlockingStub(channel);
+  }
+
+  public int takeClusterSnapshot(String clusterId) throws Exception {
+    BackupRequest backupRequest =
+        BackupRequest.newBuilder().setClusterId(clusterId).setBackupType(1).build();
```

Comment:
Could you replace all the use of hard coded value for the backup type, backup status, etc by their constants/enums ?
The code will be a bit more verbose but easier to understand and less brittle.

Reply:
Seems there are no enums for BackupType but there are for status.

Reply:
Is this one not ok `com.scalar.cassy.config.BackupType` ?

Reply:
Ah, it was in a different package.